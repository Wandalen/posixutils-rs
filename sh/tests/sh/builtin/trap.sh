trap
trap -p
trap 'a' INT
trap 'b' EXIT 6 15
trap
trap - INT
trap 'c' BUS
trap
trap 0 ABRT 15 BUS
trap
trap -- 'd' TERM ALRM
trap
