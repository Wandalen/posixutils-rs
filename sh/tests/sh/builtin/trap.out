trap -- - EXIT
trap -- - HUP
trap -- - INT
trap -- - QUIT
trap -- - ILL
trap -- - TRAP
trap -- - ABRT
trap -- - BUS
trap -- - FPE
trap -- - USR1
trap -- - SEGV
trap -- - USR2
trap -- - PIPE
trap -- - ALRM
trap -- - TERM
trap -- - CHLD
trap -- - CONT
trap -- - TSTP
trap -- - TTIN
trap -- - TTOU
trap -- - URG
trap -- - XCPU
trap -- - XFSZ
trap -- - VTALRM
trap -- - PROF
trap -- - SYS
trap -- 'b' EXIT
trap -- 'a' INT
trap -- 'b' ABRT
trap -- 'b' TERM
trap -- 'b' EXIT
trap -- 'b' ABRT
trap -- 'c' BUS
trap -- 'b' TERM
trap -- 'd' ALRM
trap -- 'd' TERM
